tests/vast-test.vala

```diff
@@ -313,6 +298,28 @@ int main (string[] args) {
         assert (e.get_value({3, 7, 29}).get_double() == 8 * 3);
         assert (e.get_value({7, 3, 29}).get_double() == 4 * 7);
 
+        var f = a.build()
+                 .tail(1, 5)
+                 .end();
+
+        assert (f.get_value({1, 1}).get_double() == 2 * 6);
+        assert (f.get_value({1, 2}).get_double() == 2 * 7);
+
+        var f1 = a.build()
+                 .qslice(1, {5}, null)
+                 .end();
+
+        assert (f1.get_value({1, 1}).get_double() == 2 * 6);
+        assert (f1.get_value({1, 2}).get_double() == 2 * 7);
+
+        var g = a.build()
+                 .index(1, 5)
```

Comment:
Just to be sure, is this conceptually equivalent to the `Array.index` utility I introduced?

https://github.com/rainwoodman/vast/blob/master/src/array.vala#L251